args.go

```diff
@@ -42,6 +43,9 @@ func newArgBuilder(opts ...Arg) (*argBuilder, error) {
 
 	var buildErr error
 	for _, opt := range opts {
+		if opt == nil {
```

Comment:
This is a good question.... my hunch is it might be unsafe to skip any nil args, as it sounds like it's probably a bug an argument was nil in the first place? i.e. Could hide issues down the line if we ignore nil args 🤔

Reply:
@briancain that sounds reasonable. I dont't have the proper context on when/how this is called and wanted to check 👍